src/System.Net.Http/src/System/Net/Http/Managed/HttpConnection.cs

```diff
@@ -209,13 +204,26 @@ private Task WriteFormattedInt32Async(int value, CancellationToken cancellationT
                     request.Headers.TransferEncodingChunked = true;
                 }
 
+                if (request.Version.Major != 1)
+                {
+                    // TODO #23134: Support 2.0
+                    throw new PlatformNotSupportedException($"Only HTTP 1.0 supported -- request.Version was {request.Version}");
```

Comment:
The resource string should be put into the .resx file (if there isn't an existing one that can be used).

Reply:
Also, the string is misleading... 1.1 is of course also supported.